appnav/src/main/kotlin/io/element/android/appnav/NotLoggedInFlowNode.kt

```diff
@@ -75,6 +77,7 @@ class NotLoggedInFlowNode @AssistedInject constructor(
         @Parcelize
         data class LoginFlow(
```

Comment:
Should not be an enum? I don't think we can have both to true, right?

Reply:
https://github.com/element-hq/element-x-android/pull/2793/commits/9f615c466f8f562a38f70935aa92090c44a93170